components/org.wso2.carbon.identity.media.core/src/main/java/org/wso2/carbon/identity/media/core/model/MediaInformation.java

```diff
@@ -26,7 +26,7 @@
 public class MediaInformation {
 
     private List<String> links;
-    private MediaInformationMetadata mediaInformationResponseMetadata;
+    private Object mediaMetadata;
```

Comment:
why we use Object here? can't we define a specific type/

Reply:
An object was used as we have decided not to process the json object read from the metadata file. All the metadata stored in the file should be returned without any processing.